msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalAuthResult.java

```diff
@@ -0,0 +1,49 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.sdk;
+
+import androidx.annotation.NonNull;
+
+import com.microsoft.identity.client.IAuthenticationResult;
+import com.microsoft.identity.client.ui.automation.sdk.AuthResult;
+
+import java.util.Map;
+
+// MSAL Result Class to handle asserting success or failure on execution of Automated Test Cases
+public class MsalAuthResult extends AuthResult {
+
+    private Map<String, ?> claims;
+
+    public MsalAuthResult(@NonNull final IAuthenticationResult authenticationResult) {
+        this.accessToken = authenticationResult.getAccessToken();
+        this.idToken = authenticationResult.getAccount().getIdToken();
+        this.userId = authenticationResult.getAccount().getId();
+        this.username = authenticationResult.getAccount().getUsername();
+        this.authority = authenticationResult.getAccount().getAuthority();
+        this.claims = authenticationResult.getAccount().getClaims();
```

Comment:
we should change the parent to make these `private` and the parent should initialize these...over here we would just call `super`

Reply:
calling super() of parent class's constructor now.